test/models.json

```diff
@@ -1,4 +1,16 @@
 [
+  {
+    "type":     "hailo",
+    "target":   "resnet_v1_18.hn",
+    "source":   "https://hailo-tappas.s3.eu-west-2.amazonaws.com/netron/resnet_v1_18.hn",
+    "format":   "HailoNN (hn)"
+  },
+  {
+    "type":     "hailo",
+    "target":   "resnet_v1_18.har",
+    "source":   "https://hailo-tappas.s3.eu-west-2.amazonaws.com/netron/resnet_v1_18.har",
```

Comment:
Is it possible to zero they weights for this model, create a smaller sized file and add upload it to a GitHub issue. The download from this location is really slow and slowing down validation runs.